README.md

```diff
@@ -1,198 +1,48 @@
-			* [P1] Make that workshop and talk available online
-			* [P1] Create two demos showing the power of our stack solving a “real problem”
-			* [P2] Determine what percentage of time source{d} engineers should spend working on docs/tutorials/talks/etc
-		* [P0] Make source{d} the lead of MLonCode research ([#42](https://github.com/src-d/okrs/issues/42))
-			* [P0] Create and maintain a research log with TL;DR summaries of research read
-			* [P1] Publish 6 ML blog posts in Q2
-			* [P2] Speak about ML on >4 relevant conferences
-			* [P2] Reach 1,000 cumulative upvotes of new blog posts on Reddit
-		* [P0] Enable sharing and publishing MLonCode models ([#44](https://github.com/src-d/okrs/issues/44))
-			* [P0] Define and design the product for model sharing (relates to
-			* [P1] Conduct user research with all key users (e.g.: key academic groups, users of our stack).
-		* [P0] Release updated content for sourced.tech ([#13](https://github.com/src-d/okrs/issues/13))
-			* [P0] Create a content strategy plan ([#35](https://github.com/src-d/okrs/issues/35))
-				* [P0] Editorial Environment Setup
-				* [P0] Customer Journeys
-				* [P0] Content Model
-				* [P0] Copywriting
-				* [P0] Metrics & Evaluation Analytics Setup
-				* [P1] Information Architecture
-				* [P2] Establish a Content Management Plan
-			* [P1] Update brand touchpoints according to content plan
-			* [P1] Apply new Visual Style to sourced.tech ([#32](https://github.com/src-d/okrs/issues/32))
-				* [P0] Visual Artifacts Study
-				* [P1] Research
-				* [P1] Chromatic Palete Study
-				* [P1] Typography Study
-				* [P1] Initial Proposals
-				* [P1] Pure css update PR
-			* [P1] Publish the CAT case-study development post on Medium ([#38](https://github.com/src-d/okrs/issues/38))
-				* [P0] Write a blog post regarding the process, methodologies and collaboration flow between teams on CAT
-	* Define DevRel organization ([#48](https://github.com/src-d/okrs/issues/48))
-		* [P0] Describe the metrics of devrel success
-		* [P0] Set monitoring up for all processes valuable to devrel
-		* [P1] Create a dashboard displaying those metrics
-		* [P0] Implement kanban processes in DevRel, follow the source{d} Engineering Methodology
-		* [P0] Set up biweekly sync-ups with Eiso, Marcelo, Maximo, and Fernanda
+# OKRs 2018 Q3
+
+## Bring 'Code as Data' to market with enterprises
+
+### gitbase & engine
+
+- [ ] Stable & Performance Release
+- [ ] Integration of gitbase with engine
+- [ ] New engine interface based on gitbase
+
+### Babelfish
+
+- [ ] Client/Library improvements, packaging, and stability
+- [ ] Iterate the higher level UAST abstractions to fix and maintain
+
+### Gemini
+
+- [ ] Release a new version with the integration of function similarity
+
+## Being a better company to work at
+
+- [ ] Regression Test, Benchmarks and Deploy platform (CD?)
+- [ ] Improve Logs, CLI, and other guides.
+- [ ] Make it easy to update and maintain PGA.
```

Comment:
I would add:

- [ ] Conduct a happiness survey and react to the unveiled issues

I think Máximo mentioned it.

Reply:
Also

- [ ] Improve the formal process of speaking at a conference 

We have nothing currently. Even a simple checklist would save us much time and nerves, especially to poor Esther who really struggled with my conferencing season. It starts with notifying her once the approval registered in src-d/conferences and ends with submitting the expenses report. It also includes what to check in the booking confirmation because I had to buy tickets at the airport one hour before the flight two times already and that sucked very much. It also includes the Visa knowledge database, e.g. British visas are tricky.

Reply:
Also

- [ ] Increase the transparency of the events happening at source{d}

Rationale: there are meetups, get togethers, holidays, beaconfellows, etc. at source{d} Many, MANY times people missed them in the calendar. Even if there are calendar records, it is so easy to forget about them. How about creating a Slack bot which will post event notifications a few (1) days before?

Reply:
Also

- [ ] Resolve the wifi connectivity problems in Madrid office

I walk with my laptop from the kitchen to my work place and it just hangs. It is not only because of Ubuntu :trollface: Anna and Esther confessed that they had the same problems. And of course MEETING ROOMS CONNECTION QUALITY ISSUES.

Reply:
>  Resolve the wifi connectivity problems in Madrid office

🙏 🙏 🙏 🙏  and has benefit of beeing a very measurable KR!

Reply:
Survey added to https://github.com/src-d/okrs/pull/71

Other proposals forwarded to Slack for the right people to take care of.